modules/secrets-store-vault.adoc

```diff
@@ -6,11 +6,20 @@
 [id="secrets-store-vault_{context}"]
 = Mounting secrets from HashiCorp Vault
 
-You can use the {secrets-store-operator} to mount secrets from HashiCorp Vault to a CSI volume in {product-title}. To mount secrets from HashiCorp Vault, your cluster must be installed either on AWS with AWS Security Token Service (STS) or on Microsoft Azure.
+You can use the {secrets-store-operator} to mount secrets from HashiCorp Vault to a CSI volume in {product-title}.
+
+[IMPORTANT]
+====
+Mounting secrets from  HashiCorp Vault by using the {secrets-store-operator} has been tested with the following cloud providers:
+
+* Amazon Web Services (AWS)
+* Microsoft Azure
+
+Other cloud providers might work, but have not been tested yet. Additional cloud providers might be tested in the future.
```

Comment:
I know we generally avoid talking about future product/integration decisions, but the language here is vague so I'll leave it up to you.

Reply:
Yeah I also am usually big on avoiding when possible, but the PM specifically asked for the last sentence because it really should work on others, we just haven't gotten around to testing them yet. But yeah leaving it vague/noncommittal on purpose